### PR TITLE
containers: replace docker.io hub with quay.io hub

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: packaging/README-dockerhub.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
```
Comment (tyll) [inline packaging/README-dockerhub.md]: This file should be rename, maybe to `README-container-registry.md`.

File: packaging/README-dockerhub.md [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
 The images are automatically rebuilt on new GIT tags or pushes to the master branch:
 
-`Dockerfile.fedora-nmstate-dev` by https://cloud.docker.com/u/nmstate/repository/docker/nmstate/fedora-nmstate-dev
-
 The base image contains a common base that is used both for the development
 image and for the distributed image.
```
Comment (tyll) [inline packaging/README-dockerhub.md]: The documentation is docker hub specific for the trigger. On Quai the default settings with the repo, container spec like `/packaging/fedora-nmstate-dev` and build context `/packaging` work. Then we get tags that are named after the branch/tag that triggered the build.

File: packaging/README-dockerhub.md (hunk 1 above)
Comment (ffmancera) [reply packaging/README-dockerhub.md]: Done, thanks!

File: packaging/README-dockerhub.md (hunk 2 above)
Comment (ffmancera) [reply packaging/README-dockerhub.md]: Done!

File: packaging/README-container-registry.md [hunk 3]
```diff
@@ -1,34 +1,18 @@
-## Docker Hub
+## Quay
 
 The images are automatically rebuilt on new GIT tags or pushes to the master branch:
```
Comment (tyll) [inline packaging/README-container-registry.md]: It is now rebuild for all branches.

File: packaging/README-container-registry.md [hunk 4]
```diff
@@ -1,34 +1,18 @@
...
 
-`Dockerfile.fedora-nmstate-dev` by https://cloud.docker.com/u/nmstate/repository/docker/nmstate/fedora-nmstate-dev
-
 The base image contains a common base that is used both for the development
```
Comment (tyll) [inline packaging/README-container-registry.md]: Uh, we do not have the build image, anymore.

File: packaging/README-container-registry.md [hunk 5]
```diff
@@ -1,34 +1,18 @@
...
-Build Context:/packaging
-Autobuild:on
-Build Caching:off
+Container spec: /packaging/fedora-nmstate-dev
+
+Build context: /packaging
+
+The tags are named after the branch/tag that triggered the build.
 
 ## Manual Building
```
Comment (tyll) [inline packaging/README-container-registry.md]: I think the source commit is not used anymore.

File: packaging/README.release.md [hunk 6]
```diff
@@ -69,7 +69,7 @@ gpg2 --verify nmstate-<version>.tar.gz.asc nmstate-<version>.tar.gz
 ```
 * Check in a clean Fedora/centOS container if the package build and install correctly.
 ```bash
-podman run -d -it --name <name> docker.io/library/fedora:31 bash
+podman run -d -it --name <name> quay.io/library/fedora:31 bash
```
Comment (tyll) [inline packaging/README.release.md]: Is this correct?

File: packaging/Dockerfile.centos8-nmstate-dev [hunk 7]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/centos:8
+FROM quay.io/library/centos:8
```
Comment (tyll) [inline packaging/Dockerfile.centos8-nmstate-dev]: Is this correct to do?

File: packaging/Dockerfile.fedora-nmstate-dev [hunk 8]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/fedora:32
+FROM quay.io/library/fedora:32
```
Comment (tyll) [inline packaging/Dockerfile.fedora-nmstate-dev]: Is this correct?

**Comment by tyll** [comment]
It seems that Travis is currently rebuilding the container image only iff /packaging is not changed, therefore this PR is not properly tested at the moment. This needs to be fixed before this PR can be merged.

File: packaging/README-container-registry.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+## Quay
+
+The images are automatically rebuilt on new GIT tags or pushes to any branch:
+
+The base image contains a common base that is used for the development
```
Comment (tyll) [inline packaging/README-container-registry.md]: The base image does not exist anymore.

File: packaging/README-container-registry.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+image.
+
+Configuration (here for `fedora-nmstate-dev`, the other build just specifies
+a different container spec (Dockerfile location):
+
+Container spec: /packaging/fedora-nmstate-dev
```
Comment (tyll) [inline packaging/README-container-registry.md]:
```suggestion
Container spec: /packaging/Dockerfile.fedora-nmstate-dev
```

File: packaging/README-container-registry.md (hunk 9 above)
Comment (tyll) [inline packaging/README-container-registry.md]:
```suggestion
```

File: packaging/README-container-registry.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+## Quay
+
+The images are automatically rebuilt on new GIT tags or pushes to any branch:
+
+The base image contains a common base that is used for the development
+image.
```
Comment (tyll) [inline packaging/README-container-registry.md]:
```suggestion
```

File: packaging/README-container-registry.md (hunk 9 above)
Comment (ffmancera) [reply packaging/README-container-registry.md]: Done, thanks!

File: packaging/README-container-registry.md (hunk 10 above)
Comment (ffmancera) [reply packaging/README-container-registry.md]: Done. thank you!

File: packaging/README-container-registry.md (hunk 9 above)
Comment (ffmancera) [reply packaging/README-container-registry.md]: Deleted, thanks!

**Pull request merged**